constants/petnums.json

```diff
@@ -2835,14 +3049,81 @@
                     30.0
                 ],
                 "statNums": {
-                    "CRIT_CHANCE": 9.428571428571429,
-                    "STRENGTH": 9.428571428571429
+                    "CRIT_CHANCE": 10.0,
+                    "STRENGTH": 10.0
                 }
             },
-            "interp": "1:64"
+            "interp": "1:100"
+        }
+    },
+    "SPIRIT": {
+        "EPIC": {
+            "1": {
+                "otherNums": [
+                    10.0
+                ],
+                "statNums": {
+                    "SPEED": 0.0,
+                    "INTELLIGENCE": 1.0
+                }
+            },
+            "100": {
+                "otherNums": [
+                    10.0
+                ],
+                "statNums": {
+                    "SPEED": 18.0,
+                    "INTELLIGENCE": 60.0
+                }
+            },
+            "interp": "1:60"
+        },
+        "LEGENDARY": {
+            "1": {
+                "otherNums": [
+                    10.0,
+                    5.5
+                ],
+                "statNums": {
+                    "SPEED": 0.0,
+                    "INTELLIGENCE": 1.0
+                }
+            },
+            "100": {
+                "otherNums": [
+                    10.0,
+                    25.2
+                ],
+                "statNums": {
+                    "SPEED": 13.0,
```

Comment:
these values are wrong and show a lvl 45 pet stats for a lvl 100 pet

Reply:
this is for the spirit pet btw

Reply:
> why was the update.json deleted? this will cause players not go get update messages if they are on older versions

I didn't know it would delete on main oops